pages/team.js

```diff
@@ -181,6 +197,8 @@ export default class Team extends Component {
         <div className='page__heading'>
           <h2>{team.name}</h2>
           { isUserModerator ? <Button variant='primary' href={`/teams/${team.id}/edit`}>Edit Team</Button> : <div /> }
+          { userId && !isMember ? <Button variant='primary' onClick={() => this.joinTeam()}>Join Team</Button> : <div /> }
```

Comment:
```suggestion
          { userId && !isMember ? <Button variant='primary' onClick={() => this.joinTeam()}>Join Team</Button> : '' }
```
Avoid rendering empty div, throwing off flex styling